apps/web/package.json

```diff
@@ -36,10 +36,9 @@
     "@remix-run/serve": "^2.7.2",
     "@remix-run/server-runtime": "^2.7.2",
     "@rollup/plugin-replace": "^5.0.5",
-    "@sentry-internal/feedback": "^7.102.1",
-    "@sentry/react": "^7.102.1",
-    "@sentry/remix": "^7.102.1",
-    "@sentry/vite-plugin": "^2.10.2",
+    "@sentry/remix": "~8.0.0-alpha.7",
+    "@sentry/vite-plugin": "^2.15.0",
+    "@sentry-internal/feedback": "~8.0.0-alpha.7",
```

Comment:
Had to add `@sentry-internal/feedback` because the modal integration is not yet exported via `@sentry/remix`. This will soon be resolved but for now we unblock this update